src/grappler/ngraph_optimizer.cc

```diff
@@ -62,15 +88,14 @@ Status NgraphOptimizer::Optimize(tensorflow::grappler::Cluster* cluster,
   }
 
   // If ngraph is disabled via ngraph_bridge api or NGRAPH_TF_DISABLE is set
-  // we will not do anything; all subsequent
-  // passes become a no-op.
+  // we will not do anything; all subsequent passes become a no-op.
   bool ngraph_not_enabled =
       (!config::IsEnabled()) || (std::getenv("NGRAPH_TF_DISABLE") != nullptr);
   bool already_processed = IsProcessedByNgraphPass(&graph);
   if (ngraph_not_enabled || already_processed) {
-    NGRAPH_VLOG(0) << "Not running through nGraph. nGraph not enabled: "
-                   << ngraph_not_enabled
-                   << " Already processed: " << already_processed;
+    NGRAPH_VLOG(0)
```

Comment:
Split this into two whether nGraph is enabled or not and then if enabled, whether we are skipping the step due to already processed. Then use VLOG level 1 to report these. Not 0. As we do anticipate workloads in which nGraph is disabled for certain parts of the computation. So it's not a Level 0 candidate.

Reply:
Sometimes we have expectation that it will run through nGraph, but it does not. Having it at Vlog0 is helpful then.  
The log message does indicate whether running through nGraph or not, using the booleans ngraph_not_enabled
and already_processed (0/1). Is that ok or we want to break it further?
```<< "NGTF_OPTIMIZER: Not running through nGraph. nGraph not enabled: "
        << ngraph_not_enabled << " Already processed: " << already_processed;```

Reply:
Only print whether we are running through nGraph finally V(0)
And V(1)

Reply:
in terms of the booleans: `already_processed` and `ngraph_not_enabled`

Will finally run thru ngraph = `already_processed || ! ngraph_not_enabled`

will finally NOT run thru ngraph: negate above: `!already_processed && ngraph_not_enabled`

will run not rewrite pass if `ngraph_not_enabled || already_processed`

will run rewrite pass (negate above line):  `!ngraph_not_enabled && !already_processed`

```
if (!already_processed && ngraph_not_enabled) {
  VLOG(0) << "Graph will not run through ngraph"
}

if (ngraph_not_enabled || already_processed) {
VLOG(1) << "Rewrite pass will not run because " + (already_processed ? "graph is already preprocessed" : "ngraph is disabled");
}

```

i'd say we can have `ngraph_enabled` instead of `ngraph_not_enabled` (and have appropriate boolean expressions after that.). 


Also, this can be in a separate PR, because this PR does not change this prints, it just shows up in the diff list because this change just adds `NGTF_OPTIMIZER` to the string